src/cromshell/__main__.py

```diff
@@ -47,6 +56,9 @@ def main_entry(verbosity):
     LOGGER.info("Invoked by: %s", " ".join(argv))
     LOGGER.info("Log level set to: %s", logging.getLevelName(logging.getLogger().level))
 
+    # Create an object to hold all cromwell configurations
+    cromwell_config.obj = VariableSetup.Cromwell()
```

Comment:
I don't know click enough to answer this question, but since we are going with a design such that all variables in the `Cromwell` class be class variables as opposed to instance variables, is such a dummy object necessary?
But on the other hand, click, by passing on the context, may **demand** such instance exists.
A question that @jonn-smith should have a better opinion of.